lib/views/help/about.html.erb

```diff
@@ -8,39 +8,37 @@
 
-	  <h2 id="how">How does it work?</h2>
-	  <p>It’s very simple: you file a request with the EU via this website. YOu email is automatically sent to the correct EU body. They have the obligation to answer within 15 working days (about three weeks).</p>
-	  <p>When the EU replies to your request, it gets published on this website and you get a notification. You get to say whether or not you are happy with the answer, to follow up for more information, or to file a confirmatory application (EU jargon meaning an appeal).</p>
-	  <p>Everyone else gets to see the correspondence and that way they don’t need to ask the same question again.</p>
-
-	  <h2 id="users">Who can use AsktheEU.org?</h2>
-	  <p>Civil society organisations, journalists, students and academics, whistleblowers and businesses might be especially interested in using the website to obtain information from the EU to carry out their work..or you might be a curious citizen! Every one of the 500 million people living in the European Union (both citizens and residents) has the legally guaranteed right to ask for documents from EU bodies.</p>
-
-	  <p><strong>Don’t delay: ask a question today!</strong></p>
-
-	  <h2 id="who">Who built AsktheEU.org?</h2>
-	  <p>AsktheEU.org is a project of <a href="http://www.access-info.org/">Access Info Europe</a>, a human rights organisation based in Madrid, and is based on <a href="http://www.mysociety.com/">mySociety</a>'s successful UK website <a href="http://WhatDoTheyKnow.com">WhatDoTheyKnow.com</a>.</p>
-	  <p>The Alaveteli software has been used in many other countries including Spain where Access Info Europe runs <a href="http://tuderechoasaber.es/">TuDerechoaSaber.es</a> together with <a href="http://Www.civio.es">Fundación Ciudadana Civio</a>.</p>
+	  <h2 id="how">Why create AsktheEU.org?</h2>
+	  <p>The European Union is an integral part of national European governance structures, and increasingly influential in every aspect of our daily lives; it has been estimated that up to 50% of national laws originate from Brussels, with the figure rising to 70% when it comes to economic policies.</p>
+	  <p>In a healthy democracy the public knows what its government is doing.</p>
+	  <p>Some information is available on EU websites and in public registers, but it is not enough. If the European public wants to hold the EU to account and to participate in decision-making, we need more information. </p>
+	  <p>At present, even though there are over 500 million EU citizens, the EU institutions receive only about 10,000 requests for documents per year.</p>
+	  <p>AsktheEU.org is designed to help members of the public find out more about what the EU is doing and to get access to information of use to you so that you can use it to engage in public debates and contribute to better decisions being taken at the EU level.</p>
 
+	  <h2 id="users">What does Access Info Europe do?</h2>
```

Comment:
I think this section loses some information about who can actually make requests, particularly the following line:

> Every one of the 500 million people living in the European Union (both citizens and residents) has the legally guaranteed right to ask for documents from EU bodies.

Worth adding that back in?